test/ably/restpush_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import six
+
+from ably import AblyRest
+
+from test.ably.restsetup import RestSetup
+from test.ably.utils import VaryByProtocolTestsMetaclass, BaseTestCase
+
+test_vars = RestSetup.get_test_vars()
+
+
+@six.add_metaclass(VaryByProtocolTestsMetaclass)
+class TestPush(BaseTestCase):
+
+    def setUp(self):
+        self.ably = AblyRest(key=test_vars["keys"][0]["key_str"],
```

Comment:
I suspect you are going to do this "init rest with testing params" quite a few times.
I suggest you make a util/helper method so that setting it up is just a one liner like this https://github.com/ably/ably-js/blob/7bec661b4c175493230f605e5355072bf7d4868a/spec/realtime/auth.test.js#L42